engine/apps/base/models/user_notification_policy_log_record.py

```diff
@@ -49,14 +49,14 @@ class UserNotificationPolicyLogRecord(models.Model):
         ERROR_NOTIFICATION_PHONE_CALLS_LIMIT_EXCEEDED,
         ERROR_NOTIFICATION_PHONE_NUMBER_IS_NOT_VERIFIED,
         ERROR_NOTIFICATION_NOT_ABLE_TO_SEND_MAIL,
-        ERROR_NOTIFICATION_MAIL_LIMIT_EXCEEDED,
-        ERROR_NOTIFICATION_EMAIL_IS_NOT_VERIFIED,
+        ERROR_NOTIFICATION_MAIL_LIMIT_EXCEEDED,  # todo: manage backend specific limits in messaging backend
```

Comment:
TODO

Reply:
Lowercase "todo" is used in our codebase too (as well as "Todo"), not sure if it's really worth it